### PR TITLE
Sigint 869: (ADO) Add PR comment support for *new* issues in Polaris

### PR DESCRIPTION
**Pull request by kishorikumar**

### CODE REVIEW

File: synopsys-task/src/synopsys-task/tools-parameter.ts [hunk 1]
```diff
@@ -80,14 +80,44 @@ export class SynopsysToolsParameter {
...
+    if (parseToBoolean(inputs.POLARIS_PR_COMMENT_ENABLED)) {
+      console.info("Polaris PR comment is enabled");
+      if (!inputs.AZURE_TOKEN) {
+        throw new Error(
+          "Missing required azure token for pull request comment"
+        );
+      }
+
+      polData.data.azure = this.setAzureData(
+        "",
```
Comment (spurohitsynopsys) [inline synopsys-task/src/synopsys-task/tools-parameter.ts:92]: Instead of setting blank values for all can we set only azure token?

Comment (kishorikumar) [reply synopsys-task/src/synopsys-task/tools-parameter.ts:92]: As we discussed, while working on other platforms' simplification, we will remove entire structure excluding azure token

**Review by kiransynopsys — APPROVED** [review]

File: synopsys-task/src/synopsys-task/model/polaris.ts [hunk 2]
```diff
@@ -12,9 +15,15 @@ export interface PolarisData {
   branch: Branch;
   project: { name: string };
   assessment: { types: string[] };
+  prcomment?: PRComment;
```
Comment (spurohitsynopsys) [inline synopsys-task/src/synopsys-task/model/polaris.ts:18]: Although bridge accepts values case insensitive, but we can make it consistent.
Current bridge mapping is polaris.prComment.enabled so we can make it as prComment

Comment (kishorikumar) [reply synopsys-task/src/synopsys-task/model/polaris.ts:18]: I believe, we are following the same pattern for other platforms as well in synopsys-extension, so we should be good.

Comment (spurohitsynopsys) [reply synopsys-task/src/synopsys-task/model/polaris.ts:18]: in action also we have prComment https://github.com/synopsys-sig/synopsys-action/blob/main/src/synopsys-action/input-data/polaris.ts#L17

Comment (kishorikumar) [reply synopsys-task/src/synopsys-task/model/polaris.ts:18]: In action it might be. Extension codebase we are following prcomment.

**Pull request merged**